lib/generators/doorkeeper/openid_connect/templates/initializer.rb

```diff
@@ -24,9 +26,12 @@
     # redirect_to new_user_session_url
   end
 
-  subject do |resource_owner|
+  subject do |resource_owner, application|
     # Example implementation:
-    # resource_owner.key
+    # resource_owner.id
+
+    # or if you need pairwise subject identifier, implement like below:
+    # Digest::SHA256.hexdigest("#{resource_owner.id}#{URI.parse(application.redirect_uri).host}#{'your_secret_salt'}")
```

Comment:
@toupeira 

> Let me know if you see another solution, otherwise I think we can merge this.

Sorry, I don't have any solution for workaround...
I think best way is implementing Registration with other parameters like `sector_identifier_uri` in the future.

>  Can you squash the commits into one, with a feature: prefix?

Squashed! 32f688b